backend-api/src/functions/asyncActiveSession/tokenService/tokenService.ts

```diff
@@ -52,6 +51,38 @@ export class TokenService implements ITokenService {
     return successResult("");
   };
 
+  private async fetchPublicKeyWithRetries(
+    stsJwksEndpoint: string,
+  ): Promise<Result<JSON>> {
+    const maxRetries = 2;
+    const delayInMs = 1000;
```

Comment:
Not looking at this in depth but I'd suggest if we were using a linear retry strategy by waiting the same amount each time, I'd recommend a 100ms delay as a starting point

Reply:
The TD states _The retries should occur in intervals of 1 second_ which is why we did the above. Are you saying you'd prefer the first retry to occur after 100ms and the second to occur after 1000ms?

Reply:
The above code is outdated but the question is still the same

Reply:
I think typo - this is an excessive amount of time to wait. Please may we update to 100ms and I'll update the TD now

Reply:
Gotcha! Will update now